src/com/ms/silverking/io/StreamUtil.java

```diff
@@ -15,265 +13,264 @@
 import com.ms.silverking.numeric.NumConversion;
 
 public final class StreamUtil {
-    private static final int    DEF_SKIP_BUF_SIZE = 4096;
-    private static final int    DEF_BUF_SIZE = 128;
-    private static final int    NEWLINE = '\n';
-    private static final int    CR = '\r';
-    
-    public static void readFully(InputStream in, byte[] buf) throws IOException {
-        readFully(in, buf, 0, buf.length);
+  private static final int DEF_SKIP_BUF_SIZE = 4096;
+  private static final int DEF_BUF_SIZE = 128;
+  private static final int NEWLINE = '\n';
+  private static final int CR = '\r';
+
+  public static void readFully(InputStream in, byte[] buf) throws IOException {
+    readFully(in, buf, 0, buf.length);
+  }
+
+  public static void readFully(InputStream in, byte[] buf, int offset, int length) throws IOException {
+    int totalBytesRead;
+
+    totalBytesRead = 0;
+    while (totalBytesRead < length) {
+      int bytesRead;
+
+      bytesRead = in.read(buf, offset + totalBytesRead, length - totalBytesRead);
+      if (bytesRead >= 0) {
+        totalBytesRead += bytesRead;
+      } else {
+        Log.warning(buf.length + " " + offset + " " + length + " " + totalBytesRead + " " + bytesRead);
+        throw new EOFException();
+      }
     }
-    
-    public static void readFully(InputStream in, byte[] buf, int offset, int length) throws IOException {
-        int    totalBytesRead;
-        
-        totalBytesRead = 0;
-        while (totalBytesRead < length) {
-            int    bytesRead;
-            
-            bytesRead = in.read(buf, offset + totalBytesRead, length - totalBytesRead);
-            if (bytesRead >= 0) {
-                totalBytesRead += bytesRead;
-            } else {
-                Log.warning(buf.length +" "+ offset +" "+ length +" "+ totalBytesRead +" "+ bytesRead);
-                throw new EOFException(); 
-            }
-        }
+  }
+
+  public static int readIntLE(InputStream in) throws IOException {
+    byte b0;
+    byte b1;
+    byte b2;
+    byte b3;
+
+    b3 = readByte(in);
+    b2 = readByte(in);
+    b1 = readByte(in);
+    b0 = readByte(in);
+    return NumConversion.bytesToInt(b0, b1, b2, b3);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 48:52, 61:65.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.